README.md

```diff
@@ -120,11 +122,12 @@ To verify `partner` and `test` pods are running:
 oc get pods -n tnf -o wide
 ```
 
-You should see something like this:
+You should see something like this (note that the test pods are running on different nodes due to a anti affinity rule):
 ```shell-script
-NAME      READY   STATUS    RESTARTS   AGE   IP           NODE       NOMINATED NODE   READINESS GATES
-partner   1/1     Running   0          22m   172.17.0.3   minikube   <none>           <none>
-test      1/1     Running   0          22m   172.17.0.4   minikube   <none>           <none>
+NAME                       READY   STATUS    RESTARTS   AGE    IP           NODE           NOMINATED NODE   READINESS GATES
+partner-68cf756959-tp2c5   1/1     Running   0          110s   10.244.1.7   minikube-m02   <none>           <none>
+test-7799cc9677-6d8qz      1/1     Running   0          110s   10.244.1.8   minikube-m02   <none>           <none>
+test-7799cc9677-rv8nv      1/1     Running   0          110s   10.244.0.5   minikube       <none>           <none>
 ```
```

Comment:
can this to be seen from workflow output logs?

Reply:
I added the log in the workflow output